### PR TITLE
`phpversion()` replacements and code branch collapsing for older PHP versions

### PR DESCRIPTION
**Pull request by Ayesh**
- Removes a `if` branch was PHP < 5, which should be safe now that we no longer support PHP 4. 
 - Replaces `phpversion()` function call with `PHP_VERSION` constant, both of which yield identical results, but the use of the constant is a micro-optimization.

### CODE REVIEW

File: mirror-info.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ function_exists('hash_file') &&
 
 echo implode('|', array(
     $MYSITE,            	// 0 : CNAME for mirror as accessed (CC, CC1, etc.)
-    phpversion(),       	// 1 : PHP version overview
+    PHP_VERSION,       	// 1 : PHP version overview
```
Comment (cmb69) [inline mirror-info.php:36]: The indentation of the comment doesn't look quite right, but it's broken anyway, so leave it as is.

File: include/ip-to-country.inc [hunk 2]
```diff
@@ -210,16 +210,9 @@ function i2c_realip()
...
-                    if (ip2long($ips[$i]) != false) {
-                        $ip = $ips[$i];
-                        break;
-                    }
-                } else {
-                    if (ip2long($ips[$i]) != -1) {
-                        $ip = $ips[$i];
-                        break;
-                    }
+                if (ip2long($ips[$i]) != false) {
```
Comment (cmb69) [inline include/ip-to-country.inc:213]: Please cater to the misleading `!= false`; either drop it altogether, or do an `!== false`.

**Review by cmb69 — COMMENTED** [review]
Thank you for the PR! Makes perfect sense, but the `!= false` should better be addressed right away.

File: include/ip-to-country.inc (hunk 2 above)
Comment (Ayesh) [reply include/ip-to-country.inc]: Good call, thank you. I dropped the `!= false` clause, and combined the `if` block with the parent.

Comment (cmb69) [reply include/ip-to-country.inc]: Ah, even better (the line is a bit long, but we can cater to that when we introduce some coding standard).

**Pull request merged**